protocol/src/main/java/org/mifos/chatbot/protocol/slack/SlackChatService.java

```diff
@@ -34,6 +34,7 @@
 import org.springframework.stereotype.Service;
 
 import javax.sql.DataSource;
+import java.io.IOException;
```

Comment:
Can we optimize imports for better readability? Basically, order and space them based on naming conventions.

Reply:
Imports are already optimized by intellij optimize import feature.